limiinal/limiinal
# Add basic website frontend

# Pull Request Template

## **Description**
This will add a React project containing source code for a basic website. Just has some info abt what Limiinal is and why its secure and all that. simple but clean imo


## **Checklist**

Please ensure your PR meets the following requirements before requesting a review:

- [n/a ] All Rust code is formatted using `cargo fmt`.
- [X] Documentation has been updated as necessary (`README.md`, code comments, etc.).

## **Testing Instructions**

1. **Steps to test this PR**:

- You may have to run 'npmi' and install the required packages including react, react-typed and react-router-dom.

2. **Expected results**:
- Should just be able to interact with the website as expected. Will be obvious if something's gone wrong.

## **Related Issues**

n/a 

## **Screenshots / Logs (if applicable)**

See discord for a video overview

## **Types of Changes**


## **Reviewer Notes**

Let me know if there's any issues or changes that could be made
---

Thank you for contributing to the Limiinal! 🎉
